finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,14 @@
+for (var i = 1; i <= 100; i++) {
```

Comment:
@m1073496 - While testing out this fizzBuzz solution, I never saw "fizzBuzz" printed out, only "fizz" or "buzz" or the number.  I would encourage you to take another pass at this exercise.

Reply:
Looking much better, @m1073496. Thanks for taking the time to update!

Reply:
Looks much better, @m1073496 - thanks for updating!